types/types.go

```diff
@@ -23,66 +24,160 @@ import (
 	"github.com/prometheus/common/model"
 )
 
+type state uint8
+
+const (
+	unprocessed state = iota
+	active
+	silenced
+	inhibited
+)
+
+// allows tracking of the status of an alert (active, silenced, )
```

Comment:
Please follow go style for comments on exported package members https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences